application/src/main/java/org/thingsboard/server/service/edge/EdgeContextComponent.java

```diff
@@ -72,49 +59,26 @@
     @Autowired
     private EdgeService edgeService;
 
-    @Autowired
-    private PartitionService partitionService;
-
     @Lazy
     @Autowired
     private EdgeEventService edgeEventService;
 
     @Lazy
```

Comment:
Instead of @Lazy on each and every dependency, you can use @Lazy where you inject the EdgeContextComponent, right?